spec.md

```diff
@@ -307,6 +310,22 @@ These limits apply for messages generated by both COs and plugins.
 | 128 bytes  | string              |
 | 4 KiB      | map<string, string> |
 
+### Error Scheme
+
+All CSI API calls defined in this spec MUST return a [standard gRPC status](https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto).
+Most gRPC libraries provide helper methods to set and read the status fields.
+
+The status `code` MUST contain a [canonical error code](https://github.com/grpc/grpc-go/blob/master/codes/codes.go). COs must handle all valid error codes. Each RPC defines a set of gRPC error codes that MUST be returned by the plugin when specified conditions are encountered. In addition to those, if the conditions defined below are encountered, the plugin MUST return the associated gRPC error code.
+
+Condition | gRPC Code | Description | Recovery Behavior
```

Comment:
Can you also include a line for the following in the current spec.
```
// Indicates that a required field is missing from the request.
// More human-readable information MAY be provided in the
// `error_description` field. The `caller_must_not_retry` field
// MUST be set to true.
//
// Recovery behavior: Caller MUST fix the request by adding the
// missing required field before retrying.
MISSING_REQUIRED_FIELD = 3;
```

I think using `INVALID_ARGUMENT` is more appropriate here because CO should not retry in this case.

Reply:
Will do.